### ISSUE TITLE
Pressing Delete Selected Entry in the Attack Animation for Item on Any Actor menu fills list with blank results

### ISSUE THREAD

**Issue by adolf14** [wrycu/StarWarsFFG-Enhancements#206]
as the title says
![Blank Results](https://github.com/wrycu/StarWarsFFG-Enhancements/assets/32248984/918dc172-c8a4-4c7a-9028-7779a9e01513)
This also appears to break all animations

**Comment by wrycu** [comment]
I'll look into fixing this. In the meantime, a GM can run the following in their console or as a macro to reset the settings:
```js
game.settings.set("ffg-star-wars-enhancements", "attack-animation-custom-entries", [
            {
                name: "grenade",
                animation: "modules/JB2A_DnD5e/Library/Generic/Impact/Impact_07_Regular_Yellow_400x400.webm",
                animation_count: "1-1",
                focus_target: false,
                sound: "modules/ffg-star-wars-enhancements/audio/explosion.wav",
            },
]);
```

**Comment by wrycu** [comment]
@adolf14 any chance you could test this fix before I merge it? Should be as simple as clicking things in the same order as you did to trigger it initially.

**Comment by adolf14** [comment]
Seems to work. Thanks for Patching that

**Comment by wrycu** [comment]
Thanks for confirming. Will be released in `2.1.1`.
